spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -265,7 +262,11 @@ def all(mac: false, xcode: false)
           end
 
           # filter out the profiles managed by xcode
-          profiles.delete_if(&:managed_by_xcode?) unless xcode
+          if xcode
+            warn('Apple API no longer returns XCode managed Provisioning Profiles')
```

Comment:
It's possible that xcode uses a param or different API for this. Needs to be investigated